.circleci/config.yml

```diff
@@ -167,6 +167,27 @@ jobs:
              export GO111MODULE=on
              go mod download
              go test ./... -v
+
+  e2e_test_aws:
+    <<: *defaults
+
+    docker:
+      - image: eversc/aws-cli
```

Comment:
It might be better to use the official `amazon/aws-cli` image, from a maintainability point of view?

Reply:
(Although then you'd need to install go...)

Reply:
Or you could do it the other way around and use an official go image and the install command from `circleci/aws-cli`?

Reply:
(Ah but then you might need to install python...) 😆

Reply:
it was a real pain with either of those options

I think it'd be good to fork my `aws-cli` to the ovotech org, another 10% task I expect as there's a bit of setup involved